content/en/docs/ambient/usage/troubleshoot-waypoint/index.md

```diff
@@ -26,14 +26,14 @@ $ kubectl exec deploy/sleep -- curl -s http://$REVIEWS_V2_POD_IP:9080/reviews/1
 Requests to the `reviews` service should be enforced by the `reviews-svc-waypoint` for any L7 policies.
 Requests to the `reviews` `v2` pod should be enforced by the `reviews-v2-pod-waypoint` for any L7 policies.
 
-1.  If your L7 policy isn't enforced, run `istioctl analyze` first to check if your policy has any validation issue.
+1.  If your L7 configuration isn't applied, run `istioctl analyze` first to check if your configuration has a validation issue.
 
     {{< text bash >}}
     $ istioctl analyze
     ✔ No validation issues found when analyzing namespace: default.
     {{< /text >}}
 
-1.  Determine which waypoint is enforcing the L7 policy for your service or pod.
+1.  Determine which waypoint is applying the L7 configuration for your service or pod.
```

Comment:
again, sort of nit picking wording that seems off to my mind, feel free to ignore
```suggestion
1.  Determine which waypoint is implementing the L7 configuration for your service or pod.
```